src/components/dev-dot-content/mdx-components/mdx-heading-permalink/index.tsx

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames'
+import { IconLink16 } from '@hashicorp/flight-icons/svg-react/link-16'
+import s from './mdx-heading-permalink.module.css'
+
+export default function MdxHeadingPermalink(props) {
+	const { className, ...rest } = props
+
+	function copyLinkToClipboard(e) {
+		navigator.clipboard.writeText(e.target.closest('a').href)
```

Comment:
We should catch any errors that might get thrown from calling `navigator.clipboard.writeText`, just in case! https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText#return_value

Reply:
@BRKalow does this look good? Open to suggestions of course!
https://github.com/hashicorp/dev-portal/pull/1089/commits/3f38ba8ce610f4d1853024a8559fb8c05446b141